sos/plugins/openstack_manila.py

```diff
@@ -43,6 +64,18 @@ def setup(self):
                 "/var/log/containers/httpd/manila-api/*log"
             ])
 
+        if self.get_option("verify"):
+            self.add_cmd_output("rpm -V %s" % ' '.join(self.packages))
```

Comment:
This is already handled in the base `Plugin` class, so it isn't needed. I know most of the other OpenStack plugins explicitly have it in there, but we have #1696 open to address that so we shouldn't add it here.

Reply:
Ack, thanks! Noticed https://github.com/sosreport/sos/blob/f2e7ac47eb96f59362c0e2c0271642377562d746/sos/plugins/__init__.py#L1373-L1384 - I'll remove this.